README.adoc

```diff
@@ -53,170 +53,122 @@ include::{common-includes}/gitclone.adoc[]
 
-Next, go to `http://localhost:9080/health` URL to see the health of the two services as well as the overall
-health outcome of the application. One check shows the state of the `system` service, and the other shows the state of
-the `inventory` service. As you might expect, both services are in the `UP` state, and the overall outcome of
-the application is `UP`.
+Visit the `http://localhost:9080/health` URL to see the health report of the two services as well as the overall
+health status of the application. One check shows the state of the `system` service, and the other shows the state of
+the `inventory` service. As you might expect, both services are in the `UP` state, and the overall health status of
+the application is `UP`:
 
-[source, role="no_copy"]
-----
-{
-  "checks": [
-      {
-        "data": {
-           "services": "available"
-        },
-        "name": "InventoryResource",
-        "state": "UP"
-      },
-      {
-        "data": {
-           "default server": "available"
-        },
-        "name": "SystemResource",
-        "state": "UP"
-      }
-   ],
-   "outcome": "UP"
-}
-----
-
-When you're done checking out the application, stop the Open Liberty server with the following
-command:
+When you are done checking out the services, stop the Open Liberty server by running the following command:
 
 ```
 mvn liberty:stop-server
 ```
 
-Navigate back to the `start` directory to begin.
-
 // =================================================================================================
 // Adding health checks to microservice
 // =================================================================================================
 
 == Adding health checks to microservices
 
-The MicroProfile Health API was added as a dependency to your `pom.xml` file. Look for the dependency with the
-`microprofile-health-api` group ID. This feature allows you to use the MicroProfile Health API to provide health checks
-for your microservices. The `mpHealth-1.0` feature is also enabled in the `src/main/liberty/config/server.xml` file.
+A health report will be generated automatically for all services that enable MicroProfile Health. This
+feature has already been enabled for you in the `src/main/liberty/config/server.xml` file:
+
+[source, xml, role="no_copy"]
+----
+<feature>mpHealth-1.0</feature>
+----
+
+All applications must provide an implementation of the `HealthCheck` interface, which will be used to
+verify the healthiness of the services.
+
 
 // =================================================================================================
 // Adding a health check to the system service
 // =================================================================================================
 
-=== Adding a health check to the system service
+=== Adding a health check to the `system` service
+
+Before you proceed, navigate to the `start` directory where you will find the starting project:
+
+```
+cd ../start
+```
```

Comment:
Is the 'cd ../start' necessary?  I can see why you removed the "Navigate back ..." sentence above now.

Reply:
Note if this change is made, this guide will be slightly different from the other ones.

Reply:
I talked about this with Jay and Laura a while back when we discussed removing the `cd start` from the "Getting started" section. Their concern was that if the users skip the "Try what you'll build" section, then they might be lost as to where to begin. So, Jay suggested to put `cd start` in the first section of the guide that talks about implementation details.

Reply:
I seem to recall this so I am fine with it as such for Health here.  We should open up a separate issue to review this comprehensively and update as appropriate across all guides.  I will get an issue created.

Reply:
Ok, sounds good.